src/AppInstallerCLICore/Workflows/MSStoreInstallerHandler.cpp

```diff
@@ -36,7 +34,7 @@ namespace AppInstaller::CLI::Workflow
             }
 
             HRESULT errorCode = S_OK;
-            context.Reporter.ExecuteWithProgress(
+            reporter.ExecuteWithProgress(

```

Comment:
Take in the `IProgressCallback`, not the reporter.  That means moving the `ExecuteWithProgress` to the caller of this.